saleor/product/views.py

```diff
@@ -84,6 +87,19 @@ def product_details(request, slug, product_id, form=None):
     return TemplateResponse(request, 'product/details.html', ctx)
 
 
+def digital_product(request, token: str) -> Union[FileResponse, HttpResponseNotFound]:
```

Comment:
This will work in the old storefront, but we'll need something similar in API/new storefront as well. We have to discuss what's the best way to achieve that.